### DISCUSSION TITLE
Jamulus iPad air

### DISCUSSION THREAD

**Discussion by oscar7480** [jamulussoftware/jamulus#1434] [Technical Support & Help] [answered]
It's possible use the jamulus software on iPad air?

**Comment by hoffie** [comment]
I think there have been some ideas and code parts floating around for iOS, but I don't think there is any installable or usable app at the moment. :(
The fully supported platforms are Windows/Linux/Mac. As of 3.7.0 there is an experimental Android APK which seems to work depending on the device (performance).

**Reply by oscar7480** [reply]
Ok, thanks

**Comment by ann0see** [comment]
Only the server works on iOS and I couldn't get the GUI to work. Basically the iOS version compiles, asks for microphone permissions and shows a black screen. So it's not useful. Unfortunately I don't have much experience with iOS programming and therefore gave up to focus on working platforms and other features (I mean we know from Android that you'll need an Ethernet cable, quality is bad,...).

The main reason I started to work on iOS support was to be able to onboard non-pc users (many probably think the same). 

Reality shows that you can get a Raspberry Pi or old PC for almost everybody, so they can participate. For me the amount of work buying Interfaces + Raspberry Pi's or even just Interfaces to skip ASIO4ALL configuration was much less than spending hours of work on iOS support which will give bad quality. People would also need to buy Ethernet adapters, interfaces etc for their iDevice. 


If anybody else can take it up, I'd really appreciate it. Please keep in mind that the GPL and the AppStore TOS might be incompatible.

**Comment by ann0see** [comment]
Update, there is a working version now. Even on the AppStore there's e.g. the KOORD app which is a heavily custimized fork but still compatible.

**Reply by rdica** [reply]
For Jamulus it seems to require typing in a hostname or IP, it doesn't scan the directory servers for server lists. Still, better than that sketchy app from an unknown source. :+1:

**Reply by ann0see** [reply]
Yes. But that's unfortunately needed.

**Reply by mcfnord** [reply]
Could a browser launch the client with ip:port as parameters? On Windows this might work with a registry entry that authorizes launching jamulus.exe from a web page. On iPad the value of this sounds much higher.

**Reply by ann0see** [reply]
I don't think so. There might be ways to support so called deep links but iOS is just very locked down. If you jailbreak and get terminal access maybe there's a way.

**Comment by admin** [comment, marked as answer]
See: https://github.com/orgs/jamulussoftware/discussions/3595 we should soon have an iOS version out.
